### PR TITLE
Release prep v30.3.0

### PR DESCRIPTION
**Pull request by AyanSinhaMahapatra**
Adds licenses from the SPDX License List version 3.23 and other licenses and updates added to Scancode.

### CODE REVIEW

**Comment by pombredanne** [comment]
Can you also add the latest licenses list?
Or should we make the licenses list a separate package?

**Comment by AyanSinhaMahapatra** [comment]
> Can you also add the latest licenses list?

Not sure what you mean here, we are adding the latest licenses list here, see commit:
https://github.com/nexB/license-expression/pull/92/commits/3fea7b41868b56aa6f94cf56c0aaa3a942a742dd

> Or should we make the licenses list a separate package?

Yes! This is related to the discussion at https://github.com/nexB/scancode-toolkit/issues/3459

We should also add changes from https://github.com/nexB/scancode-toolkit/pull/3680 here.

**Comment by pombredanne** [comment]
> Not sure what you mean here, we are adding the latest licenses list here, see commit:
3fea7b4

I had missed that! :bow:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+v30.2.1 - 2023-02-27
```
Comment (pombredanne) [inline CHANGELOG.rst:4]: Put today's date and the 30.3

Comment (AyanSinhaMahapatra) [reply CHANGELOG.rst]: Done!

**Pull request merged**